modules/pam_env/pam_env.c

```diff
@@ -53,7 +62,15 @@ typedef struct var {
 #define UNDEFINE_VAR 102
 #define ILLEGAL_VAR  103
 
+#ifdef USE_ECONF
+static int _isDirectory(const char *path);
+static char **_econf_read_file(const char *filename, const pam_handle_t *pamh,
+			       const char *delim, const char *name,
+			       const char *suffix, const char *subpath);
+#else
+static char **_read_file(const char*filename, const pam_handle_t *pamh);
```

Comment:
All identifiers starting with underscore is a reserved namespace.
The reason why we used them was static linking we used to support long time ago.
Nowadays this is just a relic from the past.

That is, you don't _have to_ introduce new identifiers from the reserved namespace.